src/components/tabs/tab-title/tab-title.spec.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import TestRenderer from 'react-test-renderer';
 import 'jest-styled-components';
```

Comment:
This import is included globally now  so can be removed https://github.com/Sage/carbon/blob/b33dd421eca039454ac53196576b8591e92eef3d/src/__spec_helper__/expect.js#L1